obspy/io/sac/sacpz.py

```diff
@@ -47,7 +47,13 @@ def _write_sacpz(inventory, file_or_file_object):
             for cha in sta:
                 resp = cha.response
                 sens = resp.instrument_sensitivity
-                paz = resp.get_paz()
+                try:
+                    paz = resp.get_paz()
+                except:
```

Comment:
Please only catch that particular exception that is being raised when no PAZ is found.

Reply:
I typically do, but it raises a base Exception.

https://github.com/obspy/obspy/blob/b3bc918ee15ff26ae173db86c997adc90154cbcc/obspy/core/inventory/response.py#L1800

Reply:
That's not the base, `BaseException` is. While `Exception` is not great, catching it is still preferable as it won't catch stuff like `KeyboardInterrupt`.